src/transformers/generation/configuration_utils.py

```diff
@@ -456,6 +456,9 @@ def __init__(self, **kwargs):
         self.num_assistant_tokens_schedule = kwargs.pop("num_assistant_tokens_schedule", "heuristic")
         self.assistant_confidence_threshold = kwargs.pop("assistant_confidence_threshold", None)
 
+        self.target_tokenizer = kwargs.pop("target_tokenizer", None)
+        self.assistant_tokenizer = kwargs.pop("assistant_tokenizer", None)
```

Comment:
Let's try to find a solution where the tokenizers are not inside the generation config. Even if it means extra kwargs in `generate` (we should avoid complex objects in the config instances)